projects/plugins/jetpack/modules/subscriptions.php

```diff
@@ -461,6 +479,22 @@ public function subscription_comment_subscribe_setting() {
 		<?php
 	}
 
+	/**
+	 * Subscribe Modal Toggle.
+	 */
+	public function subscribe_modal_setting() {
+
+		$sm_enabled = get_option( 'sm_enabled', 1 );
+		?>
+
+		<p class="description">
+			<input type="checkbox" name="sm_enabled" id="jetpack-subscribe-modal" value="1" <?php checked( $sm_enabled, 1 ); ?> />
```

Comment:
Coming back to this today; shouldn't the default / fallback here be `0` everywhere, since we do not enable the modal by default?